app/components/agenda/agendaitem/agendaitem-case-panel/agendaitem-case-panel-view.hbs

```diff
@@ -153,7 +153,22 @@
       </div>
     </Auk::Panel::Body>
   {{/if}}
-  {{#if @agendaitem.privateComment}}
+  {{#if (and (user-may "manage-agendaitems") @subcase.internalReview)}}
```

Comment:
my thought process here (and other places- was that the permission check would prevent calling the relation for other profiles when not needed. Technically we could just call the relation and it will be empty.

Reply:
Since this is a "new" place for this remark, I'm fine with the permission check here. If there are users that don't have that permission, they're used to check it in the agenda, not here.